products.d/agama-products.spec

```diff
@@ -15,18 +15,18 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 
-Name:           agama-products-opensuse
+Name:           agama-products
```

Comment:
so agama-products is just empty shell that do nothing? Do we need it at all?

Reply:
`agama-products` is the source package:

* agama-products-9+218-slfo.1.4.1.src.rpm (26.5 KB)
* agama-products-opensuse-9+218-slfo.1.4.1.noarch.rpm (22 KB)
* agama-products-sle-9+218-slfo.1.4.1.noarch.rpm (16.4 KB)